Geometry/HcalCommonData/interface/HcalDDDSimConstants.h

```diff
@@ -53,6 +53,7 @@ class HcalDDDSimConstants {
   const std::vector<HcalDetId> & getIdHF2QIE() const {return idHF2QIE;}
   double                    getLayer0Wt(const int det, const int phi, 
 					const int zside) const;
+  int                       getFrontLayer(const int det, const int eta) const;
```

Comment:
Sorry I could not find better names for this and as Kevin points out it has a different purpose and useful for Phase2 application